views/home.hbs

```diff
@@ -1,9 +1,19 @@
     <section class="container">
         <div class="my-5">
 
+              <div class="counter-wrapper">
```

Comment:
- The badge is not **Device Responsive**. The alignment of the **Website View Badge** has some minor CSS issues. Refer to the images at #15 .